src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Generated/Models/MediaJobStateChangeEventData.cs

```diff
@@ -56,15 +61,21 @@ public MediaJobStateChangeEventData(JobState previousState = default(JobState),
         /// 'Queued', 'Scheduled'
         /// </summary>
         [JsonProperty(PropertyName = "previousState")]
-        public JobState PreviousState { get; private set; }
+        public MediaJobState PreviousState { get; private set; }
```

Comment:
This could be a breaking change, will require a major version bump

Reply:
@kalyanaj Should I change this back in the swagger specification to avoid the breaking change or leave it since the prefix naming is really what we want?  I need to change the swagger for the above anyway so please let me know which way you want to go here.

Reply:
I think it is fine to have the Swagger with the new name (that includes the media prefix) (since you are GAing event grid integration only this week).

Reply:
Okay, I updated the version from 2.0.0.0 to 3.0.0.0.  I will push this change once the swagger update is merged and I can regenerate the source from the main branch instead of my fork.